pages/osx/shuf.md

```diff
@@ -0,0 +1,19 @@
+# shuf
+
+> Generate random permutations.
+
+- Randomize the order of lines in a file and output the result:
+
+`shuf {{filename}}`
+
+- Only output the first 5 entries of the result:
+
+`shuf -n {{5}} {{filename}}`
+
+- Write output to another file:
+
+`shuf -o {{filename}} {{output_filename}}`
```

Comment:
I thought this was correct with `-o {{output_file}} {{filename}}`? :)

Reply:
oh you're right